### ISSUE TITLE
Feature request: Change output device of each application

### ISSUE THREAD

**Issue by damix96** [G-Stas/Volumey#14]
I would like to be able to change the output device of each application through the UI.
same as through System -> Sound -> App volume and device settings
![image](https://user-images.githubusercontent.com/32214602/152000262-2c318d7f-2910-4adc-90cd-07da45852c4a.png)

**Comment by G-Stas** [comment]
Will research how to implement that, but I don't have much free time right now, so no ETAs.

**Comment by d-makarenko** [comment]
Yes, that would be great; I miss this feature from KDE sound controls.

**Comment by kittehzkomputerz** [comment]
I came to github looking for an answer on how to do this if it is possible in the app. I would really like to have this implemented. I hate having to go into windows settings just to move my youtube from my room speakers back to my mon without changing anything else.  I love this app for mostly keeping me out of win settings. lol. thanks!
